build-configs/common/theme/colors.ts

```diff
@@ -18,5 +19,6 @@ export const commonLightColors = {
   textDisabledColor: '#d0d0d0',
   backgroundColor: '#ffffff',
   tunewsThemeColor: '#007aa8',
-  tunewsThemeColorLight: 'rgba(0, 122, 168, 0.4)'
+  tunewsThemeColorLight: 'rgba(0, 122, 168, 0.4)',
+  poiBorderColor: '#F1F1F1'
```

Comment:
We really need another color?

Reply:
we don't have a fitting color not even close to the design. Quite too darken and the separation looks ugly. so i would say yes :)